include/admin/importers/sunlog.inc.php

```diff
@@ -240,7 +240,7 @@ function import() {
         return true;
     }
 
-    function importCategories($parentid = 0, $new_parentid = 0, $sunlogdb) {
+    function importCategories($parentid, $new_parentid, $sunlogdb) { // default: $parentid = 0, $new_parentid = 0
```

Comment:
In the other situations where the parameter without a default trailed the others, I just set a new default. Just to consider as an alternative in the future - it will tend to break less.

Reply:
Do you want to set `null` here as default? There are also 4 other places where I fixed it like that.

Reply:
Yes, that might be a good idea (the importer need work anyway, but still)

Reply:
I changed the last parameter to ´null´.